dev/rest.rst

```diff
@@ -32,9 +32,21 @@ Config Endpoints
 
 .. toctree::
    :maxdepth: 1
+   :glob:
 
    /rest/config/... <../rest/config.rst>
 
+Cluster Endpoints
+-----------------
+
+Concerns the mesh network structure.
```

Comment:
Does "mesh" add anything to "network structure"? Seems redundant to me. And all the other subtopics don't have an explanation, maybe remove this entirely making this entire section just a plain, compact listing of endpoints.

Reply:
I think an explanation for each group would be magnificent, but we can remove this one if it stands out.  Do you think the other groups could be explained in a similar manner?

Reply:
I think all the other endpoints are more or less self-explanatory. At least adding something like "Query information from database" wouldn't add much to "Database Endpoints". "Cluster Endpoints" is a bit special in that respect, as I don't think it's immediately obvious what is meant by "Cluster". So lets keep the explanation.

Reply:
In that case, I find the "mesh network" appropriate.  It's a separate class of networks and an established technical term AFAIK.